lib/utils/environment.ts

```diff
@@ -75,6 +78,7 @@ export function BuildCommand(command: string, args?: string[], options?: Command
  */
 export function ExecuteCommand(command: Command): Promise<{}> {
 	let tries = ((command.options || {}).retries || 0) + 1;
+	let delay = (command.options || {}).delay || 1000;
```

Comment:
Alternatively, `BuildCommand` could provide a default `.options` object (`{}` would help, something like `{ retries: 0, delay: 1000, ... }` with `_.defaults` would be even better), and simplify all this downstream code completely.